docs/standard/basetypes/customnumeric.md

```diff
@@ -0,0 +1,645 @@
+---
```

Comment:
This is the guidance from the material:
Including friendly easy to read URLs can improve search rankings. The URL structure, syntax and page name are all very important to the page’s ranking. The following guidelines should be followed when considering URL naming conventions.
- Include targeted keywords within the URL structure and page name to result in search engines assigning higher relevancy and in turn higher ranking. (Filenames should be a max of 80 character)
  - docs.microsoft.com/lang-locale/product-or-category-name/non-branded-keyword-relevant-page-name
  - docs.microsoft.com/en-us/azure/machine-learning-algorithm-cheat-sheet

- Search engines see “–” as a space.  These should be used to separate words in a page name:
docs.microsoft.com/en-us/azure/machine-learning-algorithm-cheat-sheet

I'd prefer if you follow the guidelines to separate the words with dashes and having friendly URL names. It's better than having to rename those later after these are published.